### ISSUE TITLE
Should we add a note for population ratio in the report?

### ISSUE THREAD

**Issue by samnawi** [inasafe/inasafe#4812]
In the report, we can see the note about displacement rate that we use to estimate the evacuated people.
![image](https://user-images.githubusercontent.com/8871958/34145458-aa7b8fa4-e4c8-11e7-9489-2c522398841b.png)

I think it will be better if the ratio that be used in InaSAFE also written in the note. 
So the user can understand how the breakdown process comes from.
Moreover, if the user just runs the analysis using raster population without aggregation, seems like there is no chance for the user to face the ratio setup in the analysis process (maybe they didn't know where the result come from once they see the breakdown information).

How about you @Charlotte-Morgan ?

**Comment by Charlotte-Morgan** [comment]
Yes - I would like the user to know the rates that were applied and if they are global defaults, rates from aggregation or counts from exposure ... and if displacement rates are default or modified
I'm not sure if that belongs here or in the provenance report

**Comment by myarjunar** [comment]
Hi @Charlotte-Morgan @samnawi we already have the rates report in the provenance report but we still can't see from where the rates is coming .

**Comment by myarjunar** [comment]
@ismailsunni currently I take the rates specification from InaSAFE defaut values from keywords, is there any ways so we can put also the source of the rates (global default, rates from aggregation, or counts from exposure) in the keywords ?

**Comment by ismailsunni** [comment]
@myarjunar 
it's quite complicated. See https://docs.google.com/spreadsheets/d/1jARdLW6cxEBk0wiUV0MJrlYVvuLseuS2zeM8prRn1dc/edit#gid=0

If it has count field in exposure layer --> it use count field
If vector exposure has no count field, but no aggregation --> it use global default
If exposure has no count field, has aggregation --> it use ratio from aggregation (can be from field or value, based on the `inasafe_default_value` or `inasafe_field` metadata)
If exposure is raster and no aggregation or with aggretion but no ratio --> global default.

Perhaps the table is clearer than my rules above.

I am not sure how will you implement this.

**Comment by Charlotte-Morgan** [comment]
I am happy if this detail is in the provenance report
